File: w08/d03/Classwork/mechanics/ex9.js
// Create a function that takes two hashes and combines the elements

hash_one = {
  "hello": "world",
  "I'm": "batman",
  "I am being": "serious"
}

hash_two = {
  "no": "really",
  "I am": "the batman",
  "legit": "dude"
}

function truth(one, two) {
    var mixedHash = {}
    for (var attrname in one) { mixedHash[attrname] = one[attrname]; }
    for (var attrname in two) { mixedHash[attrname] = two[attrname]; }
  console.log(mixedHash)
}

truth(hash_one, hash_two)

File: w09/d02/Homework/mechanics/ex9.rb
# Create a function that takes two hashes and combines the elements

hash_one = {
  "hello" => "world",
  "I'm" => "batman",
  "I am being" => "serious"
}

hash_two = {
  "no" => "really",
  "I am" => "the batman",
  "legit" => "dude"
}

def truth(one, two)
    mixed_hash = one.merge(two) {|key, first, second| first + " " + second }
  puts mixed_hash
end

truth(hash_one, hash_two)